README.md

```diff
@@ -140,6 +140,29 @@ Stream a blob into the drive at `path`. Options include
 
 Removes the `entry` at `path` from the drive. If a blob corresponding to the entry at `path` exists, it is not currently deleted.
 
+#### `watcher = db.watch([folder])`
+
+Listens to changes that are on `folder`, by default `/`.
+
+Usage example:
+```js
+for await (const [current, previous] of watcher) {
+  console.log(current.version)
+  console.log(previous.version)
```

Comment:
the snapshots here should be drive instances. we can make the drive constructor take a bee snap

Reply:
Or we make a callback on the bee watch to overwrite the current + previous, like this:
```js
return db.watch(range, { onreturn })

function onreturn (current, previous) {
  return {
    current: drive.checkout(current.version),
    previous: drive.checkout(previous.version)
  }
}
```

Reply:
just make a map function function that maps the snap, ie, map (snap) => drive.checkout(snap)